aries-site/src/examples/templates/persistent-navigation/MinimalSidebarExample.js

```diff
@@ -0,0 +1,328 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Avatar,
+  Box,
+  Button,
+  Drop,
+  Grid,
+  Header,
+  Heading,
+  List,
+  Nav,
+  ResponsiveContext,
+  Sidebar,
+  Text,
+} from 'grommet';
+import {
+  Chat,
+  Cli,
+  Clock,
+  Configure,
+  Location,
+  Hpe,
+  Projects,
+  Splits,
+  StatusInfoSmall,
+  StatusUnknown,
+} from 'grommet-icons';
+
+const pages = [
+  {
+    name: 'Focus',
+    icon: <StatusInfoSmall />,
+    content: 'panes',
+  },
+  {
+    name: 'Services',
+    icon: <Projects />,
+    content: 'cards',
+  },
+  {
+    name: 'Glances',
+    icon: <Clock />,
+    content: 'cards',
+  },
+  {
+    name: 'Flows',
+    icon: <Splits />,
+    content: 'list',
+  },
+  {
+    name: 'Locations',
+    icon: <Location />,
+    content: 'list',
+  },
+  {
+    name: 'Console',
+    icon: <Cli />,
+    content: 'panes',
+  },
+  {
+    name: 'Configure',
+    icon: <Configure />,
+    content: 'panes',
+  },
+];
+
+export const MinimalSidebarExample = ({ mobile }) => {
+  const [activeItem, setActiveItem] = React.useState(1);
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
+      <ResponsiveContext.Consumer>
+        {size => (
+          <AppContainer activeItem={activeItem}>
+            <Sidebar
+              gridArea="sidebar"
+              /* Sidebar should switch from column to row orientation 
```

Comment:
Noticing the sidebar buttons get cramped when I'm not in full screen:
![Screen Shot 2020-04-29 at 5 11 19 PM](https://user-images.githubusercontent.com/12522275/80659127-e3a2f580-8a3c-11ea-980d-bd133d280ed1.png)

Reply:
See my note in the PR description: 

> NOTE: I am having an issue with the menu items getting jumbled in the example preview mode -- Full screen looks good - I could use a fresh set of eyes on solving the jumble.

Reply:
I can look more into it as well. I looked a little last night I thought maybe it had to do with a height giving that container a `min height: 100%` or `overflow: scroll` but that was just first glance fixes.

Reply:
Those were going to be my first thoughts as well, but will have to look more as well.